doc/installation/README.md

```diff
@@ -0,0 +1,40 @@
+# Install mssql-cli
+
+## Official Installation of mssql-cli
+
+Select your platform for mssql-cli installation instructions:
+
+| [Windows](https://github.com/dbcli/mssql-cli/blob/master/doc/installation/windows.md#windows-installation) | [macOS](https://github.com/dbcli/mssql-cli/blob/master/doc/installation/macos.md#macos-installation) | [Linux](https://github.com/dbcli/mssql-cli/blob/master/doc/installation/linux.md) |
+| - | - | - |
+
+
+## Direct Downloads
+
+The instructions above are the preferred installation method. Direct downloads are provided as an alternative in the 
+scenario that your machine may not have access to the Microsoft package repository. Instructions for direct downloads can
+also be found in the links above.
+
+| Supported Platform                         |Latest Stable                 |Latest Dev               |
```

Comment:
Let's remove the latest dev column. If we ever make a fix that we want to make available, we can just point the user to the file in azure storage that has the fix.